frontend/javascripts/admin/auth/change_email_view.tsx

```diff
@@ -33,9 +35,9 @@ function ChangeEmailView() {
         window.location.href = "/auth/login";
       })
       .catch((error) => {
-        Toast.error(
-          "An unexpected error occurred while changing the email address: " + error.message,
-        );
+        const errorMsg = "An unexpected error occurred while changing the email address.";
+        Toast.error(errorMsg);
+        console.error(errorMsg, error);
```

Comment:
the messages of all `Toast.error`s are logged into the console (see frontend/javascripts/libs/toast.tsx l.210) so I will remove this again, if you dont protest :) @MichaelBuessemeyer

Reply:
nevermind, I understand you want to print the `error` too